standards/bulk.md

```diff
@@ -0,0 +1,134 @@
+# Bulk Operations
+
+## Overview
+
+The concept of bulk operations on RESTful endpoints is quite foreign, in the sense that it is not a common practice. However, there are some use cases where it makes sense to allow a client to perform multiple operations in a single request. For example, a client may want to create multiple entities at once, or update multiple entities at once. In these cases, it is often more efficient to allow the client to perform these operations in a single request, rather than making multiple requests. However, before adding support for bulk operations you should think twice if this feature is really needed. Often network performance is not what limits request throughput. Similarly, sending multiple in-parallel requests for standard REST endpoints can be a fine solution for some clients.
+
+### Synchronous
+
+Bulk operations **MUST** be synchronous when applied to an existing resource. This means that the client will receive a response to the bulk request only after all operations have been completed. This is in contrast to asynchronous operations, where the client receives a response immediately after the request is received, and then must poll for the result of the operation. Asynchronous bulk operations are applied to new resources specifically for bulk or import to enable subsequent status updates through additional endpoints.
+
+- Bulk operations **MUST** be specific to a single resource type and NOT allow for updating multiple resource types in a single request.
```

Comment:
https://github.com/SPSCommerce/wfds/tree/main/docs/endpoints#docactionbatch supports multiple action types (listed at https://github.com/SPSCommerce/wfds/tree/main/docs/endpoints#type)
The different actions are all on documents. Does that conflict with this?

Reply:
Another reference: https://github.com/SPSCommerce/network-configuration-service/blob/main/docs/api/BATCH.md

Yep this would be in conflict... we need to decide if we are looking for a bluk operations endpoint that does many resources and is generic across or is intended to be specific to a resource with a static resource definition.

Reply:
The WFDS example is all for the same resource type but does allow different actions on each entity. I don't know of a reason that we have to support that besides that it is easier to support lets endpoints.

Reply:
Thats helpful context, thanks James!